bigquery_etl/shredder/config.py

```diff
@@ -154,6 +155,7 @@ def fields(self) -> tuple[str, ...]:
 context_id_target = partial(DeleteTarget, field=CONTEXT_ID)
 
 DELETE_TARGETS: DeleteIndex = {
+    client_id_target(table="fenix_derived.new_profile_activation_v1"): FENIX_SRC,
```

Comment:
weren't there two tables here before?

Reply:
Winnie, shall we add a Jira task to add firefox_android_clients_v1 after DS-2998, so we don't forget?